pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4142,6 +4156,15 @@ int uprobe_seq_ops_trigger(struct pt_regs *ctx)
     if (data.config->tracee_pid != trigger_pid)
         return 0;
 
+    char start_text[7] = "_stext";
```

Comment:
also here - better to move this logic into a helper function, since it happens in different places.

Reply:
So something like `get_stext_addr` and `get_etext_addr`? I'll add those once other reviews come in.